core/experiments/assets/js/admin/module.js

```diff
@@ -0,0 +1,40 @@
+export default class ExperimentsModule extends elementorModules.ViewModule {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				experimentIndicators: '.e-experiment__title__indicator',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		return {
+			$experimentIndicators: jQuery( this.getSettings( 'selectors.experimentIndicators' ) ),
+		};
+	}
+
+	addTipsy( $element ) {
+		$element.tipsy( {
+			gravity: 's',
+			offset: 8,
+			title() {
+				return this.getAttribute( 'data-tooltip' );
+			},
+		} );
+	}
+
+	addIndicatorsTooltips() {
+		this.elements.$experimentIndicators.each( ( index, experimentIndicator ) => this.addTipsy( jQuery( experimentIndicator ) ) );
```

Comment:
```suggestion
		this.elements.$experimentIndicators.each( ( index, $experimentIndicator ) => this.addTipsy( $experimentIndicator  ) );
```

Reply:
The `experimentIndicator` is not a `jQuery` object